pkg/odo/cli/url/list.go

```diff
@@ -96,5 +114,7 @@ func NewCmdURLList(name, fullName string) *cobra.Command {
 		},
 	}
 	genericclioptions.AddOutputFlag(urlListCmd)
+	urlListCmd.Flags().StringVar(&o.componentContext, "context", "", "Use context to indicate the path where the component settings need to be saved and this directory should contain component source for local and binary components")
```

Comment:
please use `genericclioptions.AddContextFlag`. It was added in this PR https://github.com/openshift/odo/pull/1554/files

Reply:
added